Program.cs

```diff
@@ -22,4 +22,23 @@
     };
 });
 
+// Can be consumed by "az login --identity" by specifying MSI_ENDPOINT environment variable to this action URL
+// https://github.com/Azure/msrestazure-for-python/blob/master/msrestazure/azure_active_directory.py#L474
+
+app.MapPost("/token", async (HttpContext context, HttpRequest request) =>
+{
+    var form = await request.ReadFormAsync();
+    var resource = form["resource"].ToString();
+    var token = await tokenCredential.GetTokenAsync(new TokenRequestContext(new[] { resource }));
+    context.Response.Headers.Add("Content-Type", "application/json"); // Set the Content-Type header to JSON
```

Comment:
Not sure the comment is very useful...

Reply:
```suggestion
    context.Response.Headers.Add("Content-Type", "application/json");
```

Reply:
I agree. I saw it was a copy paste from the existing GET /token implementation and did not bother to remove it. It's done now.